### PR TITLE
Add dependency bumping doc

### PR DESCRIPTION
**Pull request by RassK**
## What

Documents which type of dependencies are used and how to bump them.

### CODE REVIEW

**Comment by RassK** [comment]
Can anyone spot if something is missing?

File: docs/internal/version-bump.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Dependency bumping
+
+## Tracked by version
+
+This section is tracking dependencies that are needed to be bumped from time to time.
```
Comment (theletterf) [inline docs/internal/version-bump.md:5]:
```suggestion
This section describes dependencies that require a periodical version bump.
```

File: docs/internal/version-bump.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+| Dependency | Files | Bumping | Notes |
+|-|-|-|-|
+| Nuget | .csproj | Upstream | Test packages might need to stay in certain version. |
```
Comment (theletterf) [inline docs/internal/version-bump.md:9]:
```suggestion
| Nuget | .csproj | Upstream | Test packages might need to stay on a certain version. |
```

File: docs/internal/version-bump.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+| Docker | *.dockerfile | Dependabot | Bumps Docker image versions |
+| dotnet SDK | (CI templates) | Manual | Search for ```actions/setup-dotnet``` or ```dotnetSdkVersion:``` |
+| ASP.NET Runtime | *.dockerfile | Manual | Search for ```./dotnet-install.sh --runtime aspnetcore``` |
+| Gihub CI OS | ./github/workflows/*.yml | Manual | Search for ```runs-on:``` |
+| APT | debian.dockerfile | Manual | Search for ```apt-get install``` |
+| Ruby gems | *.dockerfile | Manual | Search for ```gem install``` |
+
+## Tracked by checksum
+
+This section is tracking dependencies that are verified via hardcoded checksum value.
```
Comment (theletterf) [inline docs/internal/version-bump.md:20]:
```suggestion
This section describes dependencies tracked and verified using hardcoded checksum values.
```

**Review by theletterf — APPROVED** [review]

File: docs/internal/version-bump.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+This section describes dependencies that require a periodical version bump.
+
+| Dependency | Files | Bumping | Notes |
+|-|-|-|-|
+| Nuget | .csproj | Upstream | Test packages might need to stay on a certain version. |
+| Github CI | ./github/workflows/*.yml | Dependabot | Bumps Github step templates |
+| Docker | *.dockerfile | Dependabot | Bumps Docker image versions |
+| dotnet SDK | (CI templates) | Manual | Search for ```actions/setup-dotnet``` or ```dotnetSdkVersion:``` |
+| ASP.NET Runtime | *.dockerfile | Manual | Search for ```./dotnet-install.sh --runtime aspnetcore``` |
+| Gihub CI OS | ./github/workflows/*.yml | Manual | Search for ```runs-on:``` |
```
Comment (pellared) [inline docs/internal/version-bump.md:14]:
```suggestion
| GitHub CI OS | ./github/workflows/*.yml | Manual | Search for `runs-on:` |
```

File: docs/internal/version-bump.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+## Tracked by version
+
+This section describes dependencies that require a periodical version bump.
+
+| Dependency | Files | Bumping | Notes |
+|-|-|-|-|
+| Nuget | .csproj | Upstream | Test packages might need to stay on a certain version. |
+| Github CI | ./github/workflows/*.yml | Dependabot | Bumps Github step templates |
+| Docker | *.dockerfile | Dependabot | Bumps Docker image versions |
+| dotnet SDK | (CI templates) | Manual | Search for ```actions/setup-dotnet``` or ```dotnetSdkVersion:``` |
```
Comment (pellared) [inline docs/internal/version-bump.md:12]:
```suggestion
| .NET SDK | (CI templates) | Manual | Search for `actions/setup-dotnet` or `dotnetSdkVersion:` |
```

File: docs/internal/version-bump.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
+# Dependency bumping
+
+## Tracked by version
+
+This section describes dependencies that require a periodical version bump.
+
+| Dependency | Files | Bumping | Notes |
+|-|-|-|-|
+| Nuget | .csproj | Upstream | Test packages might need to stay on a certain version. |
+| Github CI | ./github/workflows/*.yml | Dependabot | Bumps Github step templates |
```
Comment (pellared) [inline docs/internal/version-bump.md:10]:
```suggestion
| GitHub CI | ./github/workflows/*.yml | Dependabot | Bumps GitHub step templates |
```

File: docs/internal/version-bump.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
+# Dependency bumping
+
+## Tracked by version
+
+This section describes dependencies that require a periodical version bump.
+
+| Dependency | Files | Bumping | Notes |
+|-|-|-|-|
+| Nuget | .csproj | Upstream | Test packages might need to stay on a certain version. |
```
Comment (pellared) [inline docs/internal/version-bump.md:9]:
```suggestion
| NuGet | .csproj | Upstream | Test packages might need to stay on a certain version. |
```

**Review by pellared — COMMENTED** [review]
one typo found and some nit casing suggestions

nice job 👍

**Review by pellared — APPROVED** [review]

**Pull request merged**